lib/noun-project-api.rb

```diff
@@ -1,10 +1,14 @@
 require 'oauth'
 require 'json'
 require 'noun-project-api/icon_retriever'
+require 'noun-project-api/reporter'
 require 'noun-project-api/icons_retriever'
 require 'noun-project-api/icon'
 
+# Top level name space for the entire Gem.
 module NounProjectApi
+  API_BASE = 'http://api.thenounproject.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.